bitnami/kubeapps/Chart.yaml

```diff
@@ -10,7 +10,7 @@ dependencies:
     version: 1.x.x
   - name: postgresql
     repository: https://charts.bitnami.com/bitnami
-    version: '10.X.X'
+    version: 10.X.X
```

Comment:
Tiny point for consistency: the common chart uses lower-case `x`, where as this one uses upper. Ignore as needed, just my brain noticing inconsistencies :)

Reply:
LOL I loved this one!  I'll apply it